src/main/java/com/microsoft/sqlserver/jdbc/SQLServerBulkCopy.java

```diff
@@ -1705,7 +1704,7 @@ private void validateStringBinaryLengths(Object colValue, int srcCol, int destCo
     }
 
     /*
-     * Retrieves the column metadata for the destination table (and saves it for later)
+     * Returns the column metadata for the destination table (and saves it for later)
      */
     private void getDestinationMetadata() throws SQLServerException {
```

Comment:
I know this is private method, but its with single asterisk, and so is considered as comment, not Javadoc. Might as well fix it here & in next method.